scripts/variantstore/wdl/GvsCreateAltAllele.wdl

```diff
@@ -11,6 +11,8 @@ workflow GvsCreateAltAllele {
     String? service_account_json_path
   }
 
+  String fq_alt_allele_table = "~{dataset_name}.alt_allele"
```

Comment:
do we want project name in this too to keep it consistent?

Reply:
yup, that turns out to be a required fix.